app/assets/locales/messages_ccodk_default.txt

```diff
@@ -28,6 +28,9 @@ install.current.version=Version currently installed: ${0}
 verify.title=CommCare Resource Verification
 verify.checking=Verifying Resources
 verify.progress=Verifying resources. ${0} resources verified of ${1} total
+verify.retry=Retry
+verify.check.message=Verifying media...

```

Comment:
I'm confused as to why we added new translations here instead of updating the text of the current ones